.github/workflows/release_docker_gateway.yaml

```
@@ -0,0 +1,43 @@
+        name: Release Gateway Docker Image
+        runs-on: ubuntu-latest
+        steps:
+            - name: Extract version from tag
+              id: extract_version
+              run: echo "::set-output name=version::${GITHUB_REF_NAME#infisical-gateway/}"
+
+            - name: Checkout code
+              uses: actions/checkout@v4
+
+            - name: 🔧 Set up QEMU
+              uses: docker/setup-qemu-action@v1
```

actionlint (1.7.4)

the runner of "docker/setup-qemu-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)